Orange/widgets/data/owimpute.py

```diff
@@ -348,6 +375,20 @@ def imputed_vars_for_column(self, column):
         else:
             assert False
 
+    def _handle_value_errors(func):
+        @wraps(func)
+        def wrapper(self, *args, **kwargs):
+            global error_shown
+            try:
+                func(self, *args, **kwargs)
+                self.error()
+                error_shown = False
+            except ValueError as ve:
```

Comment:
The exception is raised on learner inadequacy for the given data domain. Whether it _only_ happens then, I can't say with certainty. Answering this, I'm beginning to think excepting the ValueError might not be the best practice here. I'll rewrite.. thanks!!